tests/unit/coveralls.py

```diff
@@ -8,9 +8,9 @@
 # Solution provided by
 # https://stackoverflow.com/questions/32757765/conditional-commands-in-tox-tox-travis-ci-and-coveralls
 
-if __name__ == '__main__':
-    if 'COVERALLS_REPO_TOKEN' in os.environ:
-        rc = call('coveralls')
+if __name__ == "__main__":
+    if "COVERALLS_REPO_TOKEN" in os.environ:
+        rc = call("coveralls")
         sys.stdout.write("Coveralls report from TRAVIS CI.\n")
```

Comment:
```suggestion
        sys.stdout.write("Coveralls report from CI.\n")
```
We don't use Travis any more